dist/index.html

```diff
@@ -204,9 +204,16 @@
 graph TB
 	subgraph One
 		a1-->a2
-	end
+  end
   </div>
   <div class="mermaid">
+    graph TB
+      subgraph WithStyle
+        a1-->a2
+      end
+    subGraphStyle 0 stroke:#333,stroke-width:2px,rx:5px,ry:5px
```

Comment:
Syntactically I think this makes much more sense by letting subgraphs have node ids like any other node and then offering a way to style them with the same syntax: `style`, `class`, etc.

Reply:
yes, I agree, semantically as well. and that would be the same for links. I just went for the easy path by reusing the mechanism that was already there for links using their internal numerical id.

Adding id to subgraph is much larger change than "adding style to node" and would enable linking between node and subgraph and between subgraph and subgraph.

Since this is my first attempt at contributing to this code base I went with simple and easy.